### PR TITLE
gyazo: update to 1.3.1.

### PR DESCRIPTION
**Pull request by nielznl**

### CODE REVIEW

File: srcpkgs/gyazo/template
Comment (ahesford) [inline srcpkgs/gyazo/template:5]: Just remove this line, that's the default behavior.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/gyazo/template
Comment (ericonr) [inline srcpkgs/gyazo/template:9]: Are you @DirectorX ? Otherwise I'd leave it to them to fix the email.

Comment (DirectorX) [reply srcpkgs/gyazo/template]: Yes I am

Comment (nielznl) [reply srcpkgs/gyazo/template]: I have changed the maintainer to myself, however, this would be the first package for me to maintain. I am able to respond to any emails and Github notifications.

**Comment by ericonr** [comment]
Is this PR no longer valid?

**Comment by nielznl** [comment]
> Is this PR no longer valid?

Well, I was trying to contribute and thought I would start out with a version bump but didn't know how to deal with the users.noreply.github.com maintainer email

**Comment by ericonr** [comment]
The changes look ok. Maintaining a package usually just means that your opinion might have more weight on some conversation about the package, and you might be mentioned in issues about it. Some people are known to shoot emails at maintainers, as well :) 

Feel free to ping maintainers, there are a lot of PRs and it's easy to miss them. And I think you should reopen this one  c:

**Comment by nielznl** [comment]
> The changes look ok. Maintaining a package usually just means that your opinion might have more weight on some conversation about the package, and you might be mentioned in issues about it. Some people are known to shoot emails at maintainers, as well :)
> 
> Feel free to ping maintainers, there are a lot of PRs and it's easy to miss them. And I think you should reopen this one, if you want c:

Thank you for the clarification! I shouldn't have removed my fork tho, now I can not reopen this pull request. I will open a new pull request, I hope it isn't too big of a deal.

**Comment by ericonr** [comment]
Not ideal, but it's a limitation of GH.

**Pull request closed without merging**
